### PR TITLE
Feature/calendar screen

### PR DESCRIPTION
**Pull request by YaroTs**
#675 Calendar: start week from Monday
#676 Calendar: mark today
#677 Calendar: mark weekends

Week starts from Sunday only in case of USA English locale on device.

Before: 

Screenshot 2019-05-30 at 13 58 07

After: 

Screenshot 2019-05-30 at 18 21 14

### CODE REVIEW

File: mobile/src/calendar/calendar-page-weekday.tsx [hunk 1]
```diff
@@ -85,11 +87,31 @@ export class WeekDayCircle extends Component<WeekDayCircleProps> {
...
+
+            if (this.isSelectedDay(day)) {
+                return Style.color.white;
+            }
+
+            if (business.isWeekendDay(day.date)) {
+                return Style.color.gray;
+            }
+
+            if (customTextColor) {
```
Comment (antonm76) [inline mobile/src/calendar/calendar-page-weekday.tsx]: Why check for custom text color is on the last place? Shouldn't custom color win?

File: mobile/src/locale/locale.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ * Copyright (c) Arcadia, Inc. All rights reserved.
+ ******************************************************************************/
+
+import { NativeModules, Platform } from 'react-native';
+import * as moment from 'moment';
+import 'moment/locale/en-gb';
+
+//----------------------------------------------------------------------------
+export function applyLocale() {
+    const defaultLocale = 'en-gb';
```
Comment (antonm76) [inline mobile/src/locale/locale.ts]: Why 'en-gb'? Just to have some default value? How will it affect UI?

Comment (DesmanLead) [reply mobile/src/locale/locale.ts]: A week starts from Monday in GB :)

Comment (antonm76) [reply mobile/src/locale/locale.ts]: Ah, I see.

File: mobile/src/calendar/calendar-page-weekday.tsx (hunk 1 above)
Comment (YaroTs) [reply mobile/src/calendar/calendar-page-weekday.tsx]: This variable really has confusing name and actually wasn't used. I've removed in e560f680693caf5e9822d20e26502b19a80f0581

**Pull request merged**